internal/pkg/template/repository/manager/repository.go

```diff
@@ -107,7 +107,8 @@ func (r *CachedRepository) Template(ctx context.Context, reference model.Templat
 		r.d.Logger().Infof(ctx, `loading template "%s/%s"`, reference.FullName(), r.git.CommitHash())
 
 		// Load template
-		tmpl, err := loadTemplateOp.Run(ctx, r.d, r.repo, reference)
+		r.d.Logger().Infof(ctx, r.repo.Fs().BasePath())
```

Comment:
Should I use the BasePath or how the interaction with `test-projects-file` flag should work on repository level, vs template CLI Run level?